ccm_web/client/src/theme.tsx

```diff
@@ -0,0 +1,20 @@
+import { createTheme } from '@material-ui/core/styles'
+
+const ccmTheme = createTheme({
```

Comment:
I think this file doesn't need to be a `.tsx`. There's no JSX here, `.ts` should be fine.

Reply:
Renamed to `.ts`